openvdb/points/AttributeArray.h

```diff
@@ -437,8 +563,13 @@ struct UnitVecCodec
 
 
 /// Typed class for storing attribute data
+
 template<typename ValueType_, typename Codec_ = NullCodec>
+#if OPENVDB_ABI_VERSION_NUMBER >= 6 // for ABI=6, class is final to allow for de-virtualization
+class TypedAttributeArray final: public AttributeArray
```

Comment:
be good to hear if you've confirmed this breaks ABI or not as I'm curious too

Reply:
I'm still not entirely sure, my intuition is that this is implemented at the compiler optimization level so why would it affect ABI? For safety and until I learn otherwise though, I'm leaving this behind an ABI=6 macro.